internal/cli/cmd/cli.go

```diff
@@ -64,6 +65,13 @@ const (
 	cliName = "kbcli"
 )
 
+// TODO: add more commands
+var whiteList = map[string]bool{
```

Comment:
I suggest to use `cloudCmds`, only commands that used by cloud need to login first.

Reply:
ACK